analyzers/src/SonarAnalyzer.Common/SymbolicExecution/Roslyn/ExplodedNode.cs

```diff
@@ -69,7 +69,7 @@ public int AddVisit()
         && other.State.Equals(State);
 
     public override string ToString() =>
-        Operation is null
-            ? $"Block #{Block.Ordinal}, Branching{Environment.NewLine}{State}"
-            : $"Block #{Block.Ordinal}, Operation #{index}, {Operation.Instance.Serialize()}{Environment.NewLine}{State}";
+        Operation.Instance is { } operation
+            ? $"Block #{Block.Ordinal}, Operation #{index}, {operation.Serialize()}{Environment.NewLine}{State}"
+            : $"Block #{Block.Ordinal}, Branching{Environment.NewLine}{State}";
```

Comment:
[education] why replace `Operation.Instance.Serialize()` with "Branching"?

Reply:
The branch arms are swapped because the condition was negated. There is no change.

Reply:
good point!